README.md

```diff
@@ -80,7 +81,23 @@ cd lightning-terminal
 make install
 ```
 
-This will produce the `litd` executable and add it to your `GOPATH`.
+This will produce the `litd` executable and add it to your `GOPATH`. The CLI binaries for
+`lncli`, `loop`, and `frcli` will be created as well.
```

Comment:
This is only true for the `make release` command. `install` only creates the `litd` binary. Maybe we should reference the other client repos or mention to download a release build for those binaries?

Reply:
ah, darnit. Is there a reason why we do not want to create the CLI binaries on `make install`?

Reply:
I changed this text to direct them to the other repos to download the binaries.